### ISSUE TITLE
Handling #START and #END tags in files

### ISSUE THREAD

**Issue by MairusuPawa** [performous/performous#485]
Hello,

Sometimes Ultrastar files include #START and #END tags in their "header", telling the game when the song respectively begins and stops.

This was useful for bad audio rips in ancient USDX days, to remove radio announcers or unwanted intros / outros. But this has other use cases: for instance, I'm currently creating a large file for a 1-hour long metal opera, and I'm using this feature to break down the performance into chapters. Testing the file in Performous showed that the engine do not handle these tags, and they are not documented on the wiki either.

They're a discrepancy with the legacy implementation of such tags too, which has to be carried over for compatibility: #START is in seconds, while #END is in milliseconds.

Who it be possible to add support of these tags?

**Comment by alph4numb3r** [comment]
Currently the closest you can get to this behaviour is by pressing enter to skip what Performous sees as 'instrumental' (no lyrics), both at the start and the end of the song. This seems like a nice feature to have though.

**Comment by Tronic** [comment]
If these were to be supported, you must break compatibility. The one thing I've learned in software development is that keeping broken design persists for many decades but fixing the existing files and software takes mere hours at best and never more than a couple of years.

For instance, the Python programming language had already in its version 1.6 an issue with raw string literals (where you are supposed to be able to use backslashes without escaping). Due to a lexer bug, a literal like`r"c:\windows\"` won't work. This is because the lexer sees `\"` as an escaped quote mark (as it would be in normal string), and won't terminate the literal, and then raw string adds to the string a backslash and a quote mark (not only a quote mark as you might expect). It turns out that Python 3.8 still ships with the same bug, including the same documentation text used in 1.6 to describe this bug.

Lessons learned: they should have broken a few Python programs in 1990's to avoid decades of b0rkage. And they still should break a shitload of Python software to fix it for the next decades where there will be even more.

Performous should also not support broken formats, but should fix the problem and encourage others to fix their files and software.

**Comment by Lord-Kamina** [comment]
I agree with @Tronic here. OTOH, considering the scales we're talking, it would probably be trivial to have performous detect whether they're both in seconds or one in seconds and the other milliseconds.
